doc/http-api/backend_swagger.yml

```diff
@@ -1,19 +1,29 @@
 swagger: '2.0'
 info:
-  version: "0.0.0"
+  version: "1.0.0"
   title: MongooseIM control backend REST API
-  description: This REST API provides similar control functionaly to the `mongooseimctl` command. The response objects are modeled on the `ejabberd_commands` entries and print-outs from `mongooseimctl` though the final API will need to be consistent with the MongooseIM's **command registry**.
+  description: |
+    This REST API provides similar control functionaly to the `mongooseimctl` command.
+    The response objects are modeled on the `ejabberd_commands` entries and print-outs from `mongooseimctl` though the final API will need to be consistent with the MongooseIM's **command registry**.
+
+    In many cases fields like **user** or **caller** expect **JID** (jabber identity, f.e **alice@wonderland.com**).
```

Comment:
Yes, it should be **username** I changed it in models and examples but forgot to change the description...